lisette/core.py

```diff
@@ -369,29 +384,41 @@ def _trunc_str(s, mx=2000, replace="…"):
     return s[:mx]+replace if len(s)>mx else s
 
 # %% ../nbs/00_core.ipynb
+def _fmt_tool_call(o):
+    if not (c := getattr(o.choices[0].message, 'tool_calls', None)): return ''
+    calls = '\n'.join([f" `{tc.function.name}({_trunc_str(tc.function.arguments)})`" for tc in c])
```

Comment:
Ive have *some* issues in the past relying on `tc.function.name` being accessible as attributes...and instead having to access them via the `[...]` index. Sorry that I cant be more specific. But I suggest testing this one well as I've been burnt by that one in the past.